client/aliases.go

```diff
@@ -40,31 +41,21 @@ func (client *Client) performAliasAction(sa *aliasAction) (changeID string, err
 	return client.doAsync("POST", "/v2/aliases", nil, nil, bytes.NewReader(b))
 }
 
-// Alias enables the provided aliases for the snap with snapName.
-func (client *Client) Alias(snapName string, aliases []string) (changeID string, err error) {
+// Alias sets up a manual alias from alias to app in snapName.
+func (client *Client) Alias(snapName, app, alias string) (changeID string, err error) {
 	return client.performAliasAction(&aliasAction{
-		Action:  "alias",
-		Snap:    snapName,
-		Aliases: aliases,
+		Action: "alias",
+		Snap:   snapName,
+		Target: app,
+		Alias:  alias,
 	})
 }
 
-// Unalias disables explicitly the provided aliases for the snap with snapName.
-func (client *Client) Unalias(snapName string, aliases []string) (changeID string, err error) {
+// Unalias tears down a manual alias or disables all aliases of a snap (removing all manual ones)
```

Comment:
First gut feeling when seeing this is that it might feel slightly strange to mix those two very different actions. The outcome of removing a manual alias and removing all aliases for a snap with this name are pretty different, and we may end up doing one or the other depending on what happened in between (snap install --prefer, for example). Having something like `DisableAutoAliases` and `RemoveManualAlias` for those individual cases seems like a richer ground to base interfaces upon, for example. 

Even for the CLI, I'm wondering if we should really mix those two cases. Imagine the confusion of someone trying to remove the alias "foo", and it turns out that this is the main command of the snap named "foo". The command works, and the user thinks the command was removed, but actually "foo" is still there because it was not an alias itself. Instead, the _other_ aliases of the snap got removed. Uh.